pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -865,6 +865,12 @@ func (l *LibvirtDomainManager) migrateHelper(vmi *v1.VirtualMachineInstance, opt
 		}
 	}
 
+	if shouldImmediatelyFailMigration(vmi) {
```

Comment:
This seems ok to me, but lets hear what the original author of this think about this specific change.
@davidvossel , what do you think? Is this move ok?

Reply:
yup, it's fine to move this. We just need it to occur before the `dom.MigrateToURI3` function is executed.